plugins/net_plugin/net_plugin.cpp

```diff
@@ -2425,33 +2443,33 @@ namespace eosio {
 
    void net_plugin_impl::start_conn_timer(boost::asio::steady_timer::duration du, std::weak_ptr<connection> from_connection) {
       connector_check->expires_from_now( du);
-      connector_check->async_wait( [this, from_connection](boost::system::error_code ec) {
+      connector_check->async_wait(app().get_priority_queue().wrap(priority::low, [this, from_connection](boost::system::error_code ec) {
             if( !ec) {
                connection_monitor(from_connection);
             }
             else {
                elog( "Error from connection check monitor: ${m}",( "m", ec.message()));
                start_conn_timer( connector_period, std::weak_ptr<connection>());
             }
-         });
+         }));
    }
 
    void net_plugin_impl::start_txn_timer() {
       transaction_check->expires_from_now( txn_exp_period);
-      transaction_check->async_wait( [this](boost::system::error_code ec) {
+      transaction_check->async_wait(app().get_priority_queue().wrap(priority::low-1, [this](boost::system::error_code ec) {
```

Comment:
I don't really like the mixed metaphor of priority is a category and priority is a number.   I'm fine with having named points on the range, but then perhaps we can name this one as well (even if only locally).   

I guess, in a single logical bit of code it should consistently treat priority as a category or a number but not both?

I have some comments on the appbase changes too that regard this duality

Reply:
Used a variable to name it locally. appbase was updated to just always execute the highest so there are no more ranges only named points on the range.